app/export.go

```diff
@@ -83,17 +83,18 @@ func (a *App) BulkExport(writer io.Writer, outPath string, opts BulkExportOpts)
 	}
 
 	mlog.Info("Bulk export: exporting teams")
-	if err := a.exportAllTeams(writer); err != nil {
+	teamNames, err := a.exportAllTeams(writer)
+	if err != nil {
 		return err
 	}
 
 	mlog.Info("Bulk export: exporting channels")
-	if err := a.exportAllChannels(writer); err != nil {
+	if err = a.exportAllChannels(writer, teamNames); err != nil {
```

Comment:
It's good to not shadow the error again. So we can bring back the `:=` unless you specifically didn't want to?

Reply:
Without this change CI/CD was complaining. Isn't this change good so we are not shadowing the error?

Reply:
The fix is to rename `err` to `err2` so as to not shadow.

Reply:
Ah I see what you are proposing. I will keep this method for now as it is already used later on line 114:

```
	if err = a.exportAllDirectChannels(writer); err != nil {
```